### DISCUSSION TITLE
v1.4.0-beta.1

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#74] [Announcements]
- New Device: Adds support for the Asetek Hydro AIOs: H80i GT, H80i GT V2, H80i Pro, H100i GT V2, H100i GTX, H100i Pro, H110i GT V2, H110i GTX, H115i Pro, H150i Pro
  - Note: Communication with an Asetek Hydro AIO requires driver installation. Instructions are provided in `Corsair-SiUsbXpress-Driver/README.txt`.

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.4.0-beta.1'>v1.4.0-beta.1</a>.</em>

**Comment by spacegaucho** [comment]
Just installed, working well with `Corsair Hydro H100i Pro`.
